build.gradle.kts

```diff
@@ -0,0 +1,71 @@
+import net.minecraftforge.gradle.user.tweakers.ClientTweaker
+import net.minecraftforge.gradle.user.tweakers.TweakerExtension
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import org.spongepowered.asm.gradle.plugins.MixinGradlePlugin
+
+buildscript {
+    repositories {
+        maven { setUrl("http://files.minecraftforge.net/maven") }
```

Comment:
Why not use the `maven(url = "<url>")` syntax here?

Reply:
I wasn't aware of that syntax, could've sworn I've tried it before but it didn't work. I'll go ahead and change that.